test_pkm_comprehensive.py

```
@@ -201,55 +204,61 @@ def test_storage():
+        from zen.pkm.processor import ConversationProcessor
+
         # Create test config
         config = PKMConfig()
         config.auto_summarize = True
         config.extract_keywords = True
         config.generate_tags = True
         config.pkm_dir = Path("test_pkm_processor")
-        
+
         # Initialize storage and processor
         storage = PKMStorage(config)
```

Ruff (0.14.10)

Undefined name `PKMStorage`

(F821)